### PR TITLE
SOL-169: Added mobile tabs

### PR DESCRIPTION
**Pull request by ishan-torus**
## Pull request type

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

### CODE REVIEW

**Comment by metallicalfa2** [comment]
@ArpitAgrawal321 Please review this

File: src/components/WalletTabs.vue [hunk 1]
```diff
@@ -65,18 +62,18 @@ const logout = async () => {
...
         </div>
-        <div class="ml-6 flex md:hidden items-center">
+        <div v-if="selectedAddress && user.verifierId" class="ml-6 flex md:hidden items-center">
           <AccountMenuMobile><AccountMenuList :user="user" :selected-address="selectedAddress" @on-logout="logout" /></AccountMenuMobile>
         </div>
       </div>
     </nav>
 
-    <div class="py-6">
+    <div class="py-2 flex-auto overflow-y-auto w-full content-dimension">
```
Comment (ArpitAgrawal321) [inline src/components/WalletTabs.vue:76]: remove `content-dimension` , it causes weird 90% scrollbar. Replace with `py-6 sm:py-2` or something similar to add padding when mobile nav is visible.

Comment (ArpitAgrawal321) [reply src/components/WalletTabs.vue:76]: image
Current view

File: src/components/WalletTabs.vue [hunk 2]
```diff
@@ -93,10 +90,57 @@ const logout = async () => {
         </div>
       </main>
     </div>
+    <div
```
Comment (ArpitAgrawal321) [inline src/components/WalletTabs.vue:93]: Add `Position: fixed` to the mobile nav, it should be always visible. Maybe add the hide on scroll down, show on scroll up effect, not compulsory.

File: src/components/WalletTabs.vue [hunk 3]
```diff
@@ -6,15 +6,12 @@ import { useI18n } from "vue-i18n";
...
 import SolanaLightLogoURL from "@/assets/solana-light.svg";
 import { AccountMenu, AccountMenuList, AccountMenuMobile } from "@/components/nav";
-import { requireLoggedIn } from "@/modules/auth";
 import ControllerModule from "@/modules/controllers";
 import { i18n, setLocale } from "@/plugins/i18nPlugin";
 import { NAVIGATION_LIST } from "@/utils/enums";
 
 import LanguageSelector from "./nav/LanguageSelector.vue";
 
-requireLoggedIn();
```
Comment (ArpitAgrawal321) [inline src/components/WalletTabs.vue:16]: A user can access any of `/wallet` routes with redirect flow params without login. (As login checks are skipped in router for redirect flow). Not necessarily a security issue but lets keep the check here?

File: src/pages/wallet/Nft.vue [hunk 4]
```diff
@@ -60,134 +44,104 @@ const transferNFT = () => {
...
-      <div class="px-4 gt-xs:px-12 md:px-12 py-8 pb-6 relative flex flex-col items-center w-full h-full justify-center md:flex-row md:items-start">
-        <div class="flex flex-col">
-          <div class="flex justify-between px-2 py-4 w-full">
-            <div class="cursor-pointer flex items-center" @click="goBack" @keydown="goBack">
-              <ChevronLeftIcon class="text-app-text-dark-400 h-4 w-4 mr-2" />
-              <span class="text-app-text-dark-400 text-base font-light">Back</span>
-            </div>
-            <DotsHorizontalIcon class="h-6 w-6 text-app-text-dark-400 cursor-pointer" />
-          </div>
+  <WalletTabs tab="nfts" :show-header="false">
```
Comment (ArpitAgrawal321) [inline src/pages/wallet/Nft.vue:47]: image

NFT page should use entire screen width and height. `WalletTabs` add a padding-x that we don't require, we also don't need the extra space at bottom for mobile navbar for when the user is not logged in.

**Review by ArpitAgrawal321 — COMMENTED** [review]
Do we want both menu's? Hamburg(for language, acc selection) and BottomTabs(navigation) for mobile view ?

File: src/components/WalletTabs.vue (hunk 3 above)
Comment (ishan-torus) [reply src/components/WalletTabs.vue:16]: shouldn't the checks be in route(or auth service), and not in the components?
Let's get on a call for this whenever you find time.

File: src/components/WalletTabs.vue (hunk 2 above)
Comment (ishan-torus) [reply src/components/WalletTabs.vue:93]: it is visible all times, I changed the header and the footer from absolute/fixed to just block.
Arranged all three in columnar flex layout.
```
<HEAD>
<MAIN>
<FOOTER>
```

File: src/components/WalletTabs.vue (hunk 3 above)
Comment (ishan-torus) [reply src/components/WalletTabs.vue:16]: my issue is that 
1. I do not want to keep same code at 2 places [nft.vue and base.vue]
2. authentication based routing should be only done at the router.
I'll reformat this in a parallel PR.

File: src/components/WalletTabs.vue (hunk 2 above)
Comment (ArpitAgrawal321) [reply src/components/WalletTabs.vue:93]: If Main is longer than 100vh, the footer gets pushed out of the viewport. So the user has to scroll down till they find the footer.

**Comment by ishan-torus** [comment]
> 

Yup, we do.

File: src/pages/wallet/Nft.vue (hunk 4 above)
Comment (ishan-torus) [reply src/pages/wallet/Nft.vue]: Fixed

File: src/components/WalletTabs.vue (hunk 2 above)
Comment (ishan-torus) [reply src/components/WalletTabs.vue:95]: Fixed

File: src/components/WalletTabs.vue (hunk 1 above)
Comment (ishan-torus) [reply src/components/WalletTabs.vue]: fixed

**Review by ArpitAgrawal321 — APPROVED** [review]
LGTM.

**Pull request merged**